docs/sources/reference/playlist.md

```diff
@@ -11,59 +11,130 @@ weight = 4
 
 # Playlist
 
-The Playlist is a special type of Dashboard that rotates through a list of Dashboards. A Playlist can be a great way to build situational awareness, or just show off your metrics to your team or visitors.
+## About Playlists
 
-Since Grafana automatically scales Dashboards to any resolution they're perfect for big screens!
+The Playlist is a list of dashboards that are displayed in a sequence. You might use a Playlist to build situational awareness or to present your metrics to your team or visitors.
 
-## Create a playlist
-
-{{< docs-imagebox img="/img/docs/v50/playlist.png" max-width="25rem" class="docs-image--right">}}
+Grafana automatically scales Dashboards to any resolution, which makes them perfect for big screens.
 
-The Playlist feature can be accessed from Grafana's sidemenu, in the Dashboard submenu.
+You can access the Playlist feature from Grafana's side menu, in the Dashboards submenu.
 
-Click the **New playlist** button to create a new playlist. Firstly, name your playlist and configure a time interval for Grafana to wait on a particular Dashboard before advancing to the next one on the Playlist.
-
-You can search Dashboards by name (or use a regular expression), and add them to your Playlist. Or you could add tags which will include all the dashboards that belongs to a tag when the playlist start playing. By default, your starred dashboards will appear as candidates for the Playlist.
+## Create a playlist
 
-Be sure to click the **Add to playlist** next to the dashboard name to add it to the Playlist. To remove a dashboard from the playlist click on **Remove[x]** from the playlist.
+{{< docs-imagebox img="/img/docs/v50/playlist.png" max-width="25rem" class="docs-image--right">}}
 
-Since the Playlist is basically a list of Dashboards, ensure that all the Dashboards you want to appear in your Playlist are added here.
+You create a playlist to present dashboards in a sequence, with a set order and time interval between dashboards. 
+
+1. To access the Playlist feature, hover your cursor over Grafana's side menu.
+1. Click **Playlists**.
+1. Click **New playlist**.
+1. In the **Name** text box, enter a name for your playlist.
+1. In the **Interval** text box, enter a time interval.
+
+   The time interval is the amount of time for Grafana to stay on a particular dashboard before advancing to the next one on the playlist.
+1. Next to the dashboard(s) you want to add to your playlist, click **Add to playlist**. 
+1. Click **Create**.
+
+## Edit a playlist
+
+You can edit playlists while creating them or after saving them.
+
+1. To access the Playlist feature, hover your cursor over Grafana's side menu.
+1. Click **Playlists**.
+1. Click on the Playlist that you want to edit.
+
+### Edit the Name of a playlist
+   1. Double-click within the **Name** text box.
+   1. Enter a name. 
+   1. Click **Save** to save your changes.
+### Edit the Interval of a playlist
```

Comment:
```suggestion

### Edit the Interval of a playlist

```

For some reason, the the layout becomes weird unless these headers are surrounded with blank lines.

Reply:
Got it, fixed the headers so they're surrounded by blank lines.